src/binfhe/include/binfhe-constants.h

```diff
@@ -46,59 +46,51 @@ using LWEPlaintextModulus = uint64_t;
  * @brief Security levels for predefined parameter sets
  */
 enum BINFHE_PARAMSET {
```

Comment:
- Add probabilities of failure for each parameter set at the end (e.g., ; 2^(-40))
- Exclude the parameter sets with low probability of failure